### PR TITLE
feat: refactor decisions and add link to decision overlay

### PR DESCRIPTION
**Pull request by stephanwampouille**

### CODE REVIEW

**Comment by stephanwampouille** [comment]
This PR refactors our decision details since it is being used in multiple locations in the app. I added an option to represent it vertically or horizontally and added decision item hover states. I also connected the overlay to the decision item.

**Comment by stephanwampouille** [comment]
closes: #503

File: apps/detect/src/components/endpoint-overlay/endpoint-overlay.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { DarwinIcon, Overlay, ServerIcon } from "@theprelude/ds";
-import DecisionDetails from "../decision-overlay/decision-details";
+// import DecisionDetails from "../decision-overlay/decision-details";
```
Comment (kenrick) [inline apps/detect/src/components/endpoint-overlay/endpoint-overlay.tsx:2]: You can just remove comments

File: apps/detect/src/components/endpoint-overlay/endpoint-overlay.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ const EndpointOverlay = () => {
         <EndpointDetail icon={<ServerIcon />} comment={"Workstation"} />
         <EndpointDetail icon={<DarwinIcon />} comment={"MacOS"} />
       </div>
-      <DecisionDetails actions="dropdown" forEndpoint={true} />
+      {/* <DecisionDetails actions="dropdown" forEndpoint={true} /> */}
```
Comment (kenrick) [inline apps/detect/src/components/endpoint-overlay/endpoint-overlay.tsx:23]: same here

File: apps/detect/src/components/decision-overlay/decision-overlay.tsx [hunk 3]
```diff
@@ -1,16 +1,35 @@
...
+      <div className={styles.actions}>
+        <Button icon={<CrowdstrikeIcon />} intent="secondary">
+          Alert Crowdstrike
+        </Button>
+        <Button icon={<DownloadIcon />} intent="secondary">
+          Download Report
+        </Button>
+      </div>
+
+      {/* {actions === "dropdown" && (
```
Comment (kenrick) [inline apps/detect/src/components/decision-overlay/decision-overlay.tsx:26]: same here

File: packages/ds/src/components/icons/time-icon.tsx [hunk 4]
```diff
@@ -5,7 +5,7 @@ export const TimeIcon: React.FC<{ className?: string }> = ({ className }) => {
       width="16"
       height="16"
       viewBox="0 0 16 16"
-      fill="none"
+      fill="currentColor"
```
Comment (kenrick) [inline packages/ds/src/components/icons/time-icon.tsx:8]: nice! we. should create a PR where we do this for all components

**Review by kenrick — CHANGES_REQUESTED** [review]

**Comment by stephanwampouille** [comment]
@kenrick resolved comments, but lint will definitely complain. Those complaints will get resolved in through the other PR's

**Pull request merged**